src/admin/resolvers/queries/sample-queries.gql.ts

```diff
@@ -81,6 +81,15 @@ export const GET_APPROVED_ITEM_BY_URL = gql`
   ${CuratedItemData}
 `;
 
+export const GET_APPROVED_ITEM_BY_EXTERNAL_ID = gql`
+  query approvedCorpusItem($externalId: ID!) {
+    approvedCorpusItem(externalId: $externalId) {
+      ...CuratedItemData
+    }
+  }
+  ${CuratedItemData}
+`;
+
```

Comment:
It felt right to keep the `GET_` prefix in the sample query name - need an action verb here.